src/test/java/uk/gov/hmcts/reform/sendletter/services/PdfCreatorTest.java

```diff
@@ -78,6 +78,12 @@ public void should_return_a_merged_pdf_when_multiple_documents_are_passed() thro
         assertThat(actualPdfPage1).hasSameContentAs(expectedPdfPage1);
         assertThat(actualPdfPage2).hasSameContentAs(expectedPdfPage2);
 
+        actualPdfPage1.close();
```

Comment:
If we're closing those, it would be best to create them all in try-with-resources

Reply:
before it wasn't even closed, so it's just a step closer to perfection. 4 files to wrap in try with resource just to assert? dang. should fail disregarding. + it's a unit test. just closing for correct usage of stream

Reply:
What I was thinking about was that the test may fail and the streams should be closed, regardless.

Reply:
i know what you meant. would've failed before too. in a same matter ;). just adding closing statements to clear the warnings.

p.s. do you imagine 4 layered try with resources? :D

Reply:
They wouldn't be layered, you can do it in one try:

```
try (
        InputStream actualPdfPage1 = getPdfPageContents(pdfContent, 0);
        InputStream actualPdfPage2 = getPdfPageContents(pdfContent, 1);
        InputStream expectedPdfPage1 = getPdfPageContents(expectedMergedPdf, 0);
        InputStream expectedPdfPage2 = getPdfPageContents(expectedMergedPdf, 1);
) {
... assertions
}
```

Reply:
monstrosity 😂